.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: check HTML
```

Comment:
In your commit message you say that fixes #20. But I'm trying to understand how this will update the public website. What I'm missing?

Reply:
Since the scope of the issue was changed/renamed, I will accept it ;) But yes, we need to fix that.